renovate.json

```diff
@@ -99,6 +99,14 @@
       "matchStrings": ["value: \"gcr.io/cloud-devrel-public-resources/graalvm.*:(?<currentValue>.*?)\""],
       "depNameTemplate": "com.google.cloud:sdk-platform-java-config",
       "datasourceTemplate": "maven"
+    },
+    {
+      "fileMatch": [
+        "^.github/workflows/unmanaged_dependency_check.yaml$"
+      ],
+      "matchStrings": ["uses: googleapis/sdk-platform-java/java-shared-dependencies/unmanaged-dependency-check@google-cloud-shared-dependencies/v(?<currentValue>.*?)"],
+      "depNameTemplate": "com.google.cloud:sdk-platform-java-config",
```

Comment:
google-cloud-shared-dependencies ?

Reply:
This is the tag name created in sdk-platform-java: https://github.com/googleapis/sdk-platform-java/blob/main/.github/workflows/create_additional_release_tag.yaml#L26

Reply:
The shared dependencies update is merged with `com.google.cloud:sdk-platform-java-config`: https://github.com/googleapis/java-bigtable/pull/2085

Reply:
It's lucky that the two artifacts share the same version.